docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+
+services:
+  php:
+    image: spawnia/php7.1-fpm:dev
+    volumes:
+      - ./:/var/www
+    environment:
+      - APP_ENV=local
+      - APP_DEBUG=true
+      - APP_KEY=base64:w+nUc1ZQlOX4JI4PfIp1TZORaju1JfAxnsJ0SLT1A+4=
+
+      - DB_HOST=mysql
+      - DB_USERNAME=root
```

Comment:
Do we need to add `DB_NAME=lighthouse` or is that picked up somewhere else?

Reply:
The default test settings are made in DBTestCase, this just overwrites two values that are necessary for making this work with the mysql container.

Reply:
I chose to use the root user with an empty password, since that seems like the most convenient option if the need should arise to manually do something in the database.